lib/less/parser/parser.js

```diff
@@ -325,6 +325,26 @@ var Parser = function Parser(context, imports, fileInfo) {
                     }
                 },
 
+                namedColor: function () {
+                    parserInput.save();
+                    var autoCommentAbsorb = parserInput.autoCommentAbsorb;
+                    parserInput.autoCommentAbsorb = false;
```

Comment:
Never mind, I found a another workaround on top of that. Now just polishing a performance for the fix.